html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <form action="index.html">
+    <h1>Coding Form!</h1>
+    <div>Email</div>
```

Comment:
This should be the `label` element, not a `div`. This is the text your user would click to try and put focus on the input it pairs with.